app/routes/statement.js

```diff
@@ -1,31 +1,52 @@
+const Boom = require('@hapi/boom')
+
 const { getFileStream } = require('../storage')
-const joi = require('joi')
-const boom = require('@hapi/boom')
+const { get, set } = require('../cache')
+
+const streamToBuffer = require('../stream-to-buffer')
+
+const schema = require('./schemas/statement')
 
 module.exports = {
   method: 'GET',
   path: '/statement/{version}/{filename}',

```

Comment:
Is this also the correct way we should be formatting our APIs?

Should it be defined in REST?

Reply:
Redefine, updated in generator too